lib/binary/serialization.nit

```diff
@@ -383,7 +383,7 @@ redef class Char
 	do
 		v.stream.write_byte kind_char
 		# Fix when UTF-8
-		v.stream.write_byte self.ascii.to_b
+		v.stream.write_byte self.ascii
```

Comment:
This may be a problem, it could cause some programs to abort when saving something like a game with a serializable Char attribute. It's never cool to abort during serialization!

@R4PaSs could you implement writing/reading as many bytes necessary to represent UTF8 chars? or replace the soft "Fix" comment by a FIXME?